joomla/joomla-cms
# [4.0] J Installation: Install Languages. Make sure the namespace autoloader exists

### Summary of Changes
Trying to install a language during the installation routine, it requires the `JNamespacePsr4Map` class. However this class is never loaded in the installation routine.

### Testing Instructions
1. Start the Joomla installation routine
2. When Joomla is installed you get to the following page:
<img width="682" alt="image" src="https://user-images.githubusercontent.com/359377/58903359-89eb5b80-8705-11e9-80cc-438d5edfc3f8.png">
3. Click on the `Customize Installation`
4. Select the `French` language and scroll all the way down
5. Open the Developer Console (F12)
6. Open the Network tab in the Developer console
7. Click Next
8. You are now back on the page saying `Congratulations! Your Joomla site is ready.`
9. In the Network tab, click on the URL and you see this:
<img width="1439" alt="image" src="https://user-images.githubusercontent.com/359377/58903745-6d9bee80-8706-11e9-98a2-8098d2ed66d9.png">
10. Notice the text `"message":"Class 'JNamespacePsr4Map' not found"}` and the `"error":true`
11. Apply the patch
12. Click on the `Customize Installation`
13. Select the `French` language and scroll all the way down
14. Click on Next
15. You are now back on the page saying `Congratulations! Your Joomla site is ready.`
16. Click on the `Customize Installation`
17. Notice that French is no longer in the list of available languages to install. The languages has now been installed.
18. You can also check the Developer console as it now looks like this:
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/359377/58903909-ca97a480-8706-11e9-90d4-1db34d424369.png">
19. Notice the `"error":false`
20. The issue has now been fixed :)

### Expected result
The language installs without error

### Actual result
The language does not install due to an error

### Documentation Changes Required
None
